Frontend/src/app/views/dashboard/time/time.component.scss

```diff
@@ -110,44 +103,37 @@ button {
     }
   }
 
-  [nbButton].appearance-outline.size-medium,
-  .appearance-outline.size-medium[nbButtonToggle] {
+  // [nbButton].appearance-filled.size-medium,
+  .appearance-filled.size-medium[nbButtonToggle] {
     height: 100%;
     padding: 15px;
     width: 20%;
   }
 
+  // [nbButton].appearance-filled.status-basic[disabled],
+  .appearance-filled.status-basic[disabled][nbButtonToggle] {
+    width: initial !important;
+  }
+
   nb-button-group {
     width: 100%;
   }
 }
 
-.adaptive-button-text {
-  display: none;
-}
-
-.adaptive-button-icon {
-  padding: 0 !important;
-  margin: 0 !important;
-}
-
 ::ng-deep .nb-theme-default nb-card-body,
 ::ng-deep .nb-theme-default nb-card {
-  background-color: transparent;
   padding: unset !important;
   width: 100%;
-  border: unset !important;
 }
 
 /* Media query for tablet viewport targets screen size with a minimum width of 660px. */
 @media only screen and (min-width: 660px) {
   .container {
-    padding: 20px;
+    padding: 40px;
   }

```

Comment:
Can probably just remove this and use the padding that the shared `.container` gives

Reply:
Padding removed